src/commands/mirror_node.mjs

```diff
@@ -206,7 +207,53 @@ export class MirrorNodeCommand extends BaseCommand {
             rendererOptions: constants.LISTR_DEFAULT_RENDERER_OPTION
           })
         }
-      }
+      },
+      {
+        title: 'Seed DB data',
+        task: async (ctx, parentTask) => {
+          const subTasks = [
+            {
+              title: 'Insert data in public.file_data',
+              task: async (ctx, _) => {
+                const namespace = self.configManager.getFlag(flags.namespace)
+
+                const feesFileIdNum = 111;
+                const exchangeRatesFileIdNum = 112;
+                const timestamp = Date.now();
+
+                const fees = await getFileContents(this.accountManager, namespace, feesFileIdNum);
+                const exchangeRates = await getFileContents(this.accountManager, namespace, exchangeRatesFileIdNum);
+
+                const importFeesQuery = `INSERT INTO public.file_data(file_data, consensus_timestamp, entity_id, transaction_type) VALUES (decode('${fees}', 'hex'), ${timestamp + '000000'}, ${feesFileIdNum}, 17);`
+                const importExchangeRatesQuery = `INSERT INTO public.file_data(file_data, consensus_timestamp, entity_id, transaction_type) VALUES (decode('${exchangeRates}', 'hex'), ${
+                    timestamp + '000001'
+                }, ${exchangeRatesFileIdNum}, 17);`;
+                const sqlQuery = [importFeesQuery, importExchangeRatesQuery].join(`\n`);
+                const postgresPodName = 'fullstack-deployment-postgres-postgresql-0';
```

Comment:
Instead of hardcoding the pod name, you could use something like below:
```
          const pods = await this.k8.getPodsByLabel(['app.kubernetes.io/name=postgres'])
          if (pods.length !== 1) {
            throw new FullstackTestingError("postgres pod not found")
          }
          const postgresPodName = pods[0].name
```